params/protocol_params.go

```diff
@@ -35,8 +35,11 @@ const (
 	LogDataGas            uint64 = 8     // Per byte in a LOG* operation's data.
 	CallStipend           uint64 = 2300  // Free gas given at beginning of call.
 
-	Sha3Gas     uint64 = 30 // Once per SHA3 operation.
-	Sha3WordGas uint64 = 6  // Once per word of the SHA3 operation's data.
+	Sha3Gas          uint64 = 30 // Once per SHA3 operation.
+	Sha3WordGas      uint64 = 6  // Once per word of the SHA3 operation's data.
+	Keccak256Gas     uint64 = 30 // Once per KECCAK256 operation.
```

Comment:
This is not use anywhere? Do we need to add it?

Reply:
removed unused consts